sig-scalability/provider-configs.md

```diff
@@ -0,0 +1,159 @@
+
+* API server configured for load balancing, other components using standard leader election.
+
+* Etcd is used for two distinct cluster purposes - cluster state and event processing. These have different i/o characteristics. It is important to scalability testing efforts that the iops provided by the servers to etcd be consistent and protected. These leads to two requirements:
+
+    * Split etcd: Two different etcd clusters for events and cluster state (note: this is currently the GKE production default as well).
+
+    * Dedicated separate IOPS for the etcd clusters on each control plane node. On a bare metal install this could look like a dedicated SSD. This requires a more specific configuration per provider.  Config table below.
+
+<table>
+  <tr>
+    <td>Provider</td>
+    <td>Volume type</td>
+    <td>Size per etcd partition (2x)</td>
+    <td>Notes</td>
+  </tr>
+  <tr>
+    <td>Google</td>
+    <td>SSD persistent disk</td>
+    <td>256GB</td>
+    <td>Iops increases with volume size</td>
+  </tr>
+  <tr>
+    <td>AWS</td>
+    <td>EBS Provisioned IOPS SSD (io1)</td>
+    <td>256GB</td>
+    <td>Proposed</td>
+  </tr>
+  <tr>
+    <td>Azure</td>
+    <td>?</td>
+    <td>?</td>
+    <td>
+</td>
+  </tr>
+  <tr>
+    <td>Packet</td>
+    <td>Dedicated SSD</td>
+    <td>256GB</td>
+    <td>Bare metal, proposed</td>
+  </tr>
+</table>
+
+
+### Areas for Future Work
+
+* Leader election results are non-deterministic on on a typical cluster, and a config would be best served to be configured as worst-case. Not presently known whether there are performance impacts resulting from leader election resulting in either co-location or distribution of those components.
+
+* Improving the cluster performance loading to match production deployment scenarios is critical on-going work, especially clusterloader: [https://github.com/kubernetes/perf-tests/tree/master/clusterloader](https://github.com/kubernetes/perf-tests/tree/master/clusterloader)
+
+* Multi-zone / multi-az deployments are often used to manage large clusters, but for testing/scalability efforts the target is intentionally a single AZ. This keeps greater consistency between environments that do and don’t support AZ-based deployments. Failures during scalability testing are outside the SIG charter. Protecting against network partitioning and improving total cluster availability (one of the key benefits to a multi-AZ strategy) are currently out scope for the Scalability SIG efforts.
+
+* Scalability issues on very large clusters of actual nodes (instead of kubemark simulations) are real. Efforts to improve large cluster networking performance e.g. IPVS are important, and will be interesting areas for cross-SIG collaboration.
+
+### Control Plane Cluster Config 
+
+Diagram shows high level target control plan config using the server types listed above, capturing:
+
+* 5 server cluster
+
+* Split etcd across 5 nodes
+
+* API server load balanced
```

Comment:
We might want to have hints about this in here.  Like, for example, ELB, by default, has a short timeout that'll cause control plane components to resync often.  Users should set that to the max.